src/utils/tools/get-lab-config.sh

```diff
@@ -1,38 +1,62 @@
 #!/usr/bin/env bash
 
-if [[ "${USER_LAB}" == "" || "${TOKEN_LAB}" == "" ]]; then
-    echo "USER_LAB and/or TOKEN_LAB environment variables not set."
+# Check if both parameters are provided as arguments
+if [ $# -eq 2 ]; then
+    CLUSTER_NUMBER=$1
+    TOKEN=$2
+else
+    # Prompt the user for input if not provided
+    read -p "Enter cluster number: " CLUSTER_NUMBER
+    read -p "Enter token: " TOKEN
+fi
+
+# Validate that the parameters are not empty
+if [ -z "$CLUSTER_NUMBER" ] || [ -z "$TOKEN" ]; then
+    echo "Error: Both cluster-number and token must be provided."
+    echo "Usage: $0 [cluster-number] [token]"
     exit 1
 fi
 
-# download private key file
-curl -H "Authorization: token ${TOKEN_LAB}" \
+# Output the values to verify
+echo "Cluster Number: $CLUSTER_NUMBER"
+echo "Token: $TOKEN"
+
+# Check if config folder exists
+if [ ! -d "config" ]; then
+    echo "Config folder doesn't exist. Creating it..."
+    mkdir config
+fi
+
+cd config
```

Comment:
This is a bad assumption and is relative to where you invoke the script from.
I think you should do something like:
```suggestion

base_dir="$(dirname "${BASH_SOURCE[0]}")/../../../.."

# Check if config folder exists
if [ ! -d "${base_dir}/config" ]; then
    echo "Config folder doesn't exist. Creating it..."
    mkdir ${base_dir}/config
fi

cd  ${base_dir}/config
```

Reply:
the idea is to run the script anywhere, check if the `config` folder exist in the current directoy, if not, just create it, change dir inside, run the script, then return to the previous folder.